app/controllers/badge_claim_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BadgeClaimController < ApplicationController
+
+  def create
+    @claim = BadgeClaim.new
+  end
+
+  def delete
+    @claim = BadgeClaim.find(params[:claim_id])
+    @claim.delete
+  end
+
+  def redeem
+    @claim = BadgeClaim.find(params[:claim_id])
```

Comment:
Yeah, I think I was being overly cautious here. I agree, instance variables aren't needed here. Changing it.